tensorboard/webapp/webapp_data_source/tb_http_client_test.ts

```diff
@@ -103,6 +104,39 @@ describe('TBHttpClient', () => {
     });
   });
 
+  describe('XSRF header', () => {
+    it('is not attached to GET requests', () => {
```

Comment:
we tend to drop `is` as it becomes very repetitive without much information (and some reporters naturally places `is` sometimes). So, `it('attaches to non-GET requests'` would be good.

Reply:
> `it('attaches to non-GET requests'`

But “XSRF header attaches to non-GET requests” isn’t right; it’s the
client that does the attaching, not the header itself.

If you just don’t like the verb _is_, I suppose that I could write
something like “XSRF header appears on POST requests”. But “attaches to
non-GET requests” is tested in the other test cases: what we’re testing
here is that it is _not_ attached to _GET_ requests.

I don’t see a clear way to phrase that without _is not_ or _does not_,
and I’m not sure that I really understand the problem with _is_ in the
first place, so I’ve left this as is. Let me know if I misunderstand.

> some reporters naturally places `is` sometimes

That doesn’t make much sense to me, since often we put other verbs
there: e.g., “does not clobber unrelated headers” below. Any reporter
that changes “XSRF header does not […]” to “XSRF header is does not […]”
seems to broken to be worth designing around, imho.

Reply:
But often `it` refers to the subject under test. So the complete sense would be `TBHttpClient XSRF header appears on POST requests` which is very awkward because of describe in between. 

Alternative could be `TBHttpClient for XSRF attaches custom header to non-GET requests`.

Reply:
Okay: I don’t see what’s wrong with “`TBHttpClient` XSRF header appears
on POST requests” (reads like fine English to me), but I’ve taken the
structure of your suggestion in case it brings you joy. :-)